docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -165,4 +165,27 @@ writes it on the data node:
     data_retrieved = data_node.read()
     ```
 
+# Get parent scenarios, pipelines and tasks
+
+To get the parent entities of a data node, like the tasks that use the data node, or the pipeline that includes the tasks, or the scenario that contains the pipeline, you can use the `DataNode.get_parents()^` method. The method will return a dictionary with the key being the level of the parent ("scenarios", "pipelines" or "tasks") and the value is a set of the parent entities corresponding to the level. You can also use `taipy.get_parents()^`, which takes a child entity such as the data node as input, to achieve the same result.
```

Comment:
Should 'pipeline' be plural?

I don't know that I would repeat the description of what the method does.
Maybe a very short summary of it?

Reply:
I mean all the stuff after and including "The method will return...".
This can be a far simpler sentence, not repeating the RefMan content.
If it's not in the RefMan then you have a problem.

Reply:
Alright agree! The how about a shorter version:

`To get the parent entities of a data node you can either use the `DataNode.get_parents()^` or pass the data node entity to the function `taipy.get_parents()^`. Both function will return a dictionary with the keys being the level of the parents and the values is the lists of parent entities.`

Reply:
Even shorter:
"To get the parent entities of a data node you can use either the method `(DataNode.)get_parents()^` or the function 
 `taipy.get_parents()^`. Both return the parents of the data node."